tests/sync/session.cpp

```diff
@@ -335,19 +335,37 @@ TEST_CASE("sync: error handling", "[sync]") {
         CHECK(idx != std::string::npos);
         if (just_before->tm_year == just_after->tm_year) {
             std::stringstream stream;
+#if __GNUC__ < 5
+            char s[5];
+            strftime(s, 5, "%Y", just_after);
```

Comment:
Oh, come on! I'm not asking you to spend hours revising things. I'm asking you to pull out a 4 line helper function.